addons/binding/org.openhab.binding.evohome/src/main/java/org/openhab/binding/evohome/handler/EvohomeAccountBridgeHandler.java

```diff
@@ -121,8 +123,37 @@ public void setTcsMode(String tcsId, String mode) {
         tryToCall(() -> apiClient.setTcsMode(tcsId, mode));
     }
 
-    public void setPermanentSetPoint(String zoneId, double doubleValue) {
-        tryToCall(() -> apiClient.setHeatingZoneOverride(zoneId, doubleValue));
+    public void setHeatSetPoint(String zoneId, double doubleValue) {
+        if (configuration.temporaryOverrideSchedule) {
+            try {
```

Comment:
This is a rather large block. Try to reduce the scope of try catch blocks as much as possible. If I recall correctly, you don't need it at all; tryToCall solves that for you.